interface/patient_file/report/custom_report.php

```diff
@@ -189,25 +189,25 @@ function zip_content($source, $destination, $content = '', $create = true)
       // can also be used by other forms that require output in the
       // encounter listings output, but not in the custom report. ?>
 <style>
-  div.navigateLink {display:none;}
-  .hilite {background-color: var(--yellow);}
-  .hilite2 {background-color: transparent;}
-  mark {background-color: var(--yellow);}
-  .btn {
-      cursor:pointer;
+    div.navigateLink {
+        display:none;
     }
-  .next {
-      background-color: var(--yellow);
+
+    .hilite {
+        background-color: var(--yellow);
+    }
+    .hilite2 {
+        background-color: transparent;
+    }
+    mark {
+        background-color: var(--yellow);
+    }
+    .next {
+        background-color: var(--yellow);
+    }
+    img {
+        max-width:700px;
     }
```

Comment:
Hi  @tywrenn  and @bradymiller 

I Guess these classes are outdated, need your check, Should I remove it?

Reply:
Yeah a lot of these are outdated and should be removed. For the one's involving a yellow highlight they can just simply have the `bg-warning` class tacked onto them.